src/logic/frozen.js

```diff
@@ -66,11 +66,10 @@ Frozen.prototype.create = async function (data, trs) {
     const senderId = data.sender.address;
     const airdropReward = await self.getAirdropReward(senderId, data.freezedAmount, data.type);
 
-    const date = new Date(trs.timestamp * 1000);
     trs.recipientId = null;
     trs.asset.stakeOrder = {
         stakedAmount: data.freezedAmount,
-        nextVoteMilestone: (date.setMinutes(date.getMinutes())) / 1000,
+        nextVoteMilestone: trs.timestamp,
```

Comment:
Is it correct milestone? Why not `trs.timestamp + constants.froze.vTime * 60`?

Reply:
correct, because it`s create. After creation you can vote and get first voteCount increment